### ISSUE TITLE
When compression was done from a stream, the output file after decompression is called -

### ISSUE THREAD

**Issue by karel-brinda** [kowallus/mbgc#7]

**Comment by kowallus** [comment]
Yes, it is. _mbgc_ is tailored to work with collection of genomes.

**Comment by kowallus** [comment]
I should rephrase. mbgc is tailored to work with a collection of FASTA files. 
I understand the problem. Maybe specifying an output path in decompression would make handling output file '-' more readable.

**Comment by karel-brinda** [comment]
I've actually tried this and specified `seq.fa`, and the output got into `seq.fa/-`, which I first even didn't notice and thought the directory was empty.

**Comment by kowallus** [comment]
What about decompressing to stdout and redirecting to a specified file?
`mbgc d spn01.mbgc - > stream.fasta`

**Comment by karel-brinda** [comment]
That's what I'm doing now (after all of the experiments), but it's not obvious for users this is the way to go.

**Comment by kowallus** [comment]
I would suggest such an approach. I will later add appropriate comments to the documentation.
